### DISCUSSION TITLE
【求助】多刷新了几下就成这样了…

### DISCUSSION THREAD

**Discussion by muxmus** [share121/inter-knot#68] [其他]
<img width="500" alt="" src="https://github.com/user-attachments/assets/7d998a80-8787-4443-8374-16ff2c6ce62c">
<img width="500" alt="" src="https://github.com/user-attachments/assets/fe15db62-eb2e-492e-8a84-14890caa4166">

@share121 

**绳网被玩坏了！**

**这肯定不是绳网的问题！**
**绝对不是！**

**Comment by share121** [comment]
你刷新的太快了

**Comment by DeepslateQAQ** [comment]
有考虑给接口setInterval或setTimeout吗 懒得手动刷新了@share121

**Comment by muxmus** [comment]
我就点了一次刷新，然后就加载不出来了，网址后面的`/?code=***`一直在变，是不是因为我禁用了第三方cookie，待会儿取消了再试试

**Reply by Mu-7593** [reply]
> 我就点了一次刷新，然后就加载不出来了，网址后面的`/?core=***`一直在变，是不是因为我禁用了第三方cookie，待会儿取消了再试试

网址开始变化时按停止加载此页后稍等一会，就正常登录了好像

**Reply by muxmus** [reply]
感谢！按照你的方法已经可以了
